src/decorator/decoratorConfig.ts

```diff
@@ -38,7 +38,7 @@ const getEnvironment = (): Environment => {
   } else if (erDev()) {
     return "q2";
   } else {
-    return "local";
+    return "mock";
```

Comment:
For å bli kvitt feilmeldingen om tilkobling til contextholderen man ser øverst ved kjøring lokalt.